regulations/static/regulations/css/less/typography.less

```diff
@@ -42,35 +42,50 @@ h3,
 h4,
 h5,
 h6 {
-    .font-bold;
-    margin-bottom: 0;
+    .sans-font-bold;
+    margin: 0.5em 0;
 }
 
 h1 {
-    font-size: 2.125em;
+  font-weight: 300;
+  font-size: 44px;
+  line-height: 50px;
 }
 
-h2, h3.section-title {
-    font-size: 1.625em;
+h2 {
+  font-weight: 300;
+  font-size: 44px;
+  line-height: 50px;
 }
 
 h3 {
-    font-size: 1.375em;
+  font-size: 34px;
+  line-height: 40px;
 }
 
 h4 {
-    font-size: 1em;
-    text-transform: uppercase;
+  font-size: 24px;
+  line-height: 29px;
+  text-transform: none;
```

Comment:
yeah I think you're right. I copied that over from notice-and-comment and it probably just had the `none` because it was overwriting the `uppercase` that I just replaced.